Sources/Crypto/Key Agreement/ECDH.swift

```diff
@@ -88,6 +88,14 @@ extension P256 {
                 let pemDocument = ASN1.PEMDocument(type: "PUBLIC KEY", derBytes: self.derRepresentation)
                 return pemDocument.pemString
             }
+			
+			public static func ==(lhs: Self, rhs: Self) -> Bool {
```

Comment:
Equatable, Part of #174 but I was unable to create a diff between just this PR and the source branch of that PR.